packages/reference/src/common/SingleReferenceEditor.tsx

```diff
@@ -81,7 +81,7 @@ export function SingleReferenceEditor(
           <Editor
             {...props}
             key={`${externalReset}-reference`}
-            entityId={value ? value.sys.id : ''}
+            entityId={value ? value?.sys?.id : ''}
```

Comment:
If you want this work down the road, let's say in a couple of months, as folks might wonder why this is required and might change it back, consider adding a unit test for your use case.